manifests/plugin/openscap.pp

```diff
@@ -0,0 +1,66 @@
+# = Foreman Openscap plugin
+#
+# This class installs openscap plugin
+#
+# === Parameters:
+#
+# $configure_openscap_repo::  Enable custom yum repo with packages needed for foreman_openscap,
+#                             type:boolean
+#
+# $scap_client_module_dir::   if you set a directory here, installer will upload foreman_scap_client
+#                             puppet module into this directory for you
+#                             e.g. /etc/puppet/environments/production/modules
+#
+class foreman::plugin::openscap (
+  $configure_openscap_repo = false,
+  $scap_client_module_dir  = undef,
+) {
+  case $::osfamily {
+    'RedHat': {
+      validate_bool($configure_openscap_repo)
+
+      if $configure_openscap_repo {
+        case $::operatingsystem {
+          'fedora': {
+            $repo = 'fedora'
+          }
+          default: {
+            $repo = 'epel'
+          }
+        }
+
+        if $::osreleasemajor == undef {
+          if $::operatingsystemmajrelease == undef {
+            $versions_array = split($::operatingsystemrelease, '\.') # facter 1.6
+            $major = $versions_array[0]
+          } else {
+            $major = $::operatingsystemmajrelease # facter 2.2
+          }
+        } else {
+          $major = $::osreleasemajor
+        }
+
+        yumrepo { 'isimluk-openscap':
```

Comment:
Fair enough. So the policy is plugins in our repos, core packages preferably in base OS.